documentation-site/pages/blog/style-updates/index.mdx

```diff
@@ -0,0 +1,495 @@
+import {withStyle} from 'baseui';
+import {Block} from 'baseui/block';
+import {
+  Table,
+  StyledTable,
+  StyledHead,
+  StyledHeadCell,
+  StyledBody,
+  StyledRow,
+  StyledCell,
+} from 'baseui/table';
+
+import metadata from './metadata.json';
+import Layout from '../../../components/layout';
+import {BlogImage, Meta, Caption} from '../../../components/blog';
+
+export default Layout;
+
+<Meta data={metadata} />
+
+<BlogImage
+  src="/static/images/blog/style-updates/baseblack.png"
+  alt="updated base web components with black highlights"
+  caption="Base Web has an updated look"
+/>
+
+## Migration Guide
+
+After bumping to `v8.17`, your app will look a little different. Grays will be darker and type will be larger. Input highlights and buttons will be black rather than blue. _Don't panic!_ If you address each of the points below your app will look right in no time!
+
+- Replace direct usage of gray scale (`$theme.colors.mono100` for example) with new values. Make sure to look for references in `Block` as well. [Read more 📖](#light-theme-mapping)
+- Replace direct usage of typographic scale (`$theme.typography.font100` for example) with new values. Make sure to look for references in `Block` as well. [Read more 📖](#typography-mapping)
```

Comment:
do we want people to replace these values? if we think it's a must, should we release as a major?

Reply:
+1 I think this should be a major version bump. It's a breaking change unless manual migration (i.e. follow the upgrade guide) is done.